.github/workflows/deploy-lambda-function.yml

```
@@ -186,7 +186,7 @@ jobs:
       # deploy
       - name: 'Deploy'
         id: deploy
-        uses: shopsmart/github-actions/actions/deploy-lambda-function@v3
+        uses: shopsmart/github-actions/actions/deploy-lambda-function@hotfix/sleep-before-publish-version
```

```suggestion
        uses: shopsmart/github-actions/actions/deploy-lambda-function@v3
```

This will become the new v3